insights/2014-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,146 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsClient",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/{resourceUri}/usages": {
+      "get": {
+        "tags": [
+          "UsageMetrics"
+        ],
+        "operationId": "UsageMetrics_List",
+        "description": "The List operation lists the usage metrics for the resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2014-04-01",
```

Comment:
As discussed with Andy, he asked us to make this a required parameter and remove the default value. Can you make this change and regenerate the .net and node.js code